openshift/e2e-common.sh

```diff
@@ -111,7 +111,19 @@ function run_e2e_rekt_tests(){
   echo "Replacing knative-eventing-test-eventshub image"
   sed -i -e "s|registry.ci.openshift.org/openshift/knative-.*:knative-eventing-test-eventshub|${KNATIVE_EVENTING_TEST_EVENTSHUB}|g" "$(dirname "$0")/../vendor/knative.dev/reconciler-test/pkg/eventshub/103-pod.yaml"
 
-  go_test_e2e -timeout=1h -parallel=20 ./test/rekt || failed=$?
+  local test_name="${1:-}"
+  local run_command=""
+  local failed=0
+ 
+  if [ -n "$test_name" ]; then
+      local run_command="-run ^(${test_name})$"
+  fi
+  # check for test flags
+  RUN_FLAGS=("-timeout=90m -parallel=20")
```

Comment:
You're using an array of one element here. And if EVENTING_TEST_FLAGS is defined the value will overwrite the single element in the array, two lines below.
Not sure it's intentional, the array is useless here but it works. Using just plain string would be ok as well:
```
RUN_FLAGS="-timeout=50m -parallel=20"
if [ -n "${EVENTING_TEST_FLAGS:-}" ]; then
  RUN_FLAGS="${EVENTING_TEST_FLAGS}"
fi
...
...
go_test_e2e ${RUN_FLAGS} ./test/rekt
```

Reply:
So you want me replace the array to string value everywhere in the code?

Reply:
Yes, please. On lines that you modified.